src/Build/BackEnd/Components/RequestBuilder/TaskHost.cs

```diff
@@ -364,6 +370,12 @@ public void Reacquire()
             // to release explicitly granted cores when reacquiring the node may lead to deadlocks.
             ReleaseAllCores();
 
+            // If file accesses are being reported yielding is a no-op so reacquire should be too.
```

Comment:
Why are we putting this after ReleaseAllCores? Seems like if we are skipping Reacquire, we should fully skip it...

Also, wouldn't skipping yield and reacquire have rather bad perf implications, potentially?

Reply:
`RequestCores` (and `ReleaseCores`) are separate calls which are made. Although a caller *shouldn't* do this, they may rely on all the requested cores to be released on `Reacquire`. So we still want that behavior.